source/adminguide/templates.rst

```diff
@@ -295,6 +295,57 @@ Example GUI dialog of uploading Template/ISO from local (browser) is given below
 
 Note that uploading multi-disk templates is also supported.
 
+Sharing templates with other accounts/projects
+----------------------------------------------
+
+When adding a template, the owner can choose to make template public or to keep it private. Once the template is created, the owner can choose to share this template so that other accounts/projects can also use the template. 
+
+Currently, the template owner can share his template with:
+  - other accounts inside his own domain (i.e. can't share the template with other accounts in the subdomain of his domain or any other domains)
+  - projects where he belongs to (i.e. if he is the owner/creator or projects of other users where he has been joined)
```

Comment:
the text in brackets is not clear.

(i.e. if he is the owner/creator of XXXX?, or to projects of other users where he has been joined)?

Reply:
resolved as suggested, thx